include/revng/ABI/ApplyRegisterStateDeductions.h

```diff
@@ -0,0 +1,263 @@
+#pragma once
```

Comment:
Rename to `RegisterStateDeductions.{h,.cpp}`.

Reply:
Done (https://github.com/revng/revng/pull/98/commits/3be0a383f482727d8da1bbc82485cc329f2761b6)